charts/linkerd-control-plane/values.yaml

```diff
@@ -525,6 +525,9 @@ nodeSelector:
 # for more information
 #nodeAffinity:
 
+# -- url of external prometheus instance
```

Comment:
nitpicky but should we indicate this is used for the heartbeat?

Reply:
would you like I add this comment ?